src/main/java/com/marklogic/client/dataloader/impl/DefaultSchemasDataFinder.java

```diff
@@ -0,0 +1,29 @@
+package com.marklogic.client.dataloader.impl;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.marklogic.client.dataloader.SchemasDataFinder;
+
+/**
+ * Finds all files in a directory.
+ * TODO only select xsd, tde, and ruleset?
+ *
+ */
+public class DefaultSchemasDataFinder implements SchemasDataFinder {
+
+	private String schemasPath = "data/schemas";
+	
+	@Override
+	public List<File> findSchemasData(File schemasBase) {
```

Comment:
Can you rename "schemasBase" to "baseDir"? That'll keep it consistent with DefaultModulesFinder.